lib/cisco_node_utils/syslog_settings.rb

```diff
@@ -42,6 +42,68 @@ def ==(other)
       name == other.name
     end
 
+    def console
```

Comment:
You should add default getters here for `console, monitor and source_interface` like you did for radius_global.